bungee/src/main/resources/bungee.yml

```diff
@@ -9,6 +9,7 @@ version: ${project.version}-${git.commit.id.abbrev}
 author: games647, https://github.com/games647/FastLogin/graphs/contributors
 
 softDepends:
+    - floodgate
```

Comment:
Looks like I've added this in another commit, so it didn't get reverted.
I'll keep it here since it should have been added in another Pull Request anyways.

Reply:
Then please include it in the other commit. Commit should be atomic. Otherwise it gets chaotic if we cherry-pick or revert certain commits. You could use the rebase functionality to override an existing commit.

Reply:
I have rebased it as you have requested. And don't worry, you are not strict in any ways, the changes you've asked for so far were all totally reasonable.